admin-default.properties

```diff
@@ -236,17 +237,17 @@ LOST_RID_API=${mosip.regproc.status.service.url}/registrationprocessor/v1/regist
 logging.level.org.springframework.web.client.RestTemplate=DEBUG
 
 # Roles
-mosip.role.admin.getpacketstatusupdate=REGISTRATION_ADMIN
-mosip.role.admin.postbulkupload=GLOBAL_ADMIN,REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.getbulkuploadtranscationtranscationid=GLOBAL_ADMIN,REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.getbulkuploadgetalltransactions=GLOBAL_ADMIN,REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.getpacketstatusupdate=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.postbulkupload=GLOBAL_ADMIN
+mosip.role.admin.getbulkuploadtranscationtranscationid=GLOBAL_ADMIN
+mosip.role.admin.getbulkuploadgetalltransactions=GLOBAL_ADMIN
 mosip.role.admin.postauditmanagerlog=GLOBAL_ADMIN,ZONAL_ADMIN
 
 
-mosip.role.admin.getgeneratecsrcertificateapplicationidreferenceid=KEY_MAKER
-mosip.role.admin.postuploadcertificate=KEY_MAKER
-mosip.role.admin.postgeneratecsr=KEY_MAKER
-mosip.role.admin.postuploadotherdomaincertificate=KEY_MAKER
+mosip.role.admin.getgeneratecsrcertificateapplicationidreferenceid=GLOBAL_ADMIN,ZONAL_ADMIN
```

Comment:
seems like roles are not upto date in qa-1201-b2 branch. 
FYI @syedsalman3753  https://github.com/mosip/mosip-config/blob/develop-v3/admin-default.properties#L246
Existing collab values are valid. No need to override it here.